docs/faucet.md

```diff
@@ -0,0 +1,90 @@
+# Faucet Service
+This service can be used to fund accounts with kin  
+The main repository is at [github.com/kinecosystem/stellar-faucet](https://github.com/kinecosystem/stellar-faucet)
+
+# Set up the faucet service  
+## Prerequisites
+
+
+* Edit the 'seeds.txt' file with the seeds and channels you wish to use.  
+  The main seed is mandatory, and all of the kin funded will be taken from this account.  
```

Comment:
+  Replace with > The main seed is mandatory since all of the funded KIN will be sent from this account.